tiledb/dataframe_.py

```diff
@@ -524,7 +525,8 @@ def from_pandas(uri, dataframe, **kwargs):
     mode = tiledb_args.get("mode", "ingest")
     attr_filters = tiledb_args.get("attr_filters", True)
     dim_filters = tiledb_args.get("dim_filters", True)
-    coords_filters = tiledb_args.get("coords_filters", None)
+    coords_filters = tiledb_args.get("coords_filters", True)
+    offsets_filters = tiledb_args.get("offsets_filters", True)
```

Comment:
Not part of this PR but as a future TODO refactoring: remove the duplication of defaults between these `.get()` calls and `TILEDB_KWARG_DEFAULTS`